### DISCUSSION TITLE
table组件强烈建议增加此功能！td内可以有类似tableselect插件的功能，可以点击td表格弹窗可以跟API交互弹窗，弹窗内容为分页数据或是表格数据

### DISCUSSION THREAD

**Discussion by 383387267qqcom** [layui/layui#1859] [建议]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

最新

### 问题描述

table组件强烈建议增加此功能！td内可以有类似tableselect插件的功能，可以点击td表格弹窗可以跟API交互弹窗，弹窗内容为分页数据或是表格数据。
1：满足表格选择大批量数据时，查看选择功能。
2：满足表格选择时，大批量数据的筛选功能。
3：满足轻松或是简单的方式就可以集成进去。而不是现在东拼西凑
4：强烈建议官方再出一个独立的类似tableselect的插件，并持续的优化和更新

### 业务代码

```auto
参考tableselect插件，社区开发的
```

### 浏览器

最新版本

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by Sight-wcg** [comment]
建议用 table 和 layer 根据需求自行封装。

**Comment by Hgqin** [comment]
https://dev.layuion.com/extend/dropdownTable/
建议你用我开发的，那个没更新过，我这个就是基于2.8.0写的

**Comment by hjingsuper** [comment]
参考：[https://github.com/layui/layui/issues/1506#issuecomment-1869407083](url)
在openSub1弹窗中渲染table，点击【确定】后执行回调，或许能实现要求

**Comment by sentsim** [comment]
抛开建议内容先不谈，我觉得 Issue 的**标题还可以再长一点** 😅
